crates/construction/src/manufacturing.rs

```diff
@@ -306,6 +314,18 @@ impl ProductionItem {
     }
 }
 
+fn setup(mut commands: Commands, server: Res<AssetServer>) {
```

Comment:
I would encapsulate as much sound logic as possible to `de_audio` crate. This should include loading the assets and playing the sound. The interface could be via events like this: 

```rust
audio.send(PlayAudioEvent::new(Sound::Manufacture, location));
```